cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -99,6 +99,9 @@ def get_short_id_from_lb2_arn(alb_arn: str) -> str:
     'elasticmapreduce:cluster': {'label': 'EMRCluster', 'property': 'arn'},
     'es:domain': {'label': 'ESDomain', 'property': 'arn'},
     'kms:key': {'label': 'KMSKey', 'property': 'arn'},
+    'iam:group': {'label': 'AWSGroup', 'property': 'arn'},
```

Comment:
It'll take a bit more work to support tagged policies since cartography has an [unusual way](https://github.com/lyft/cartography/blob/c14ef09e74f7269c1c0fb4482aaf5bba6281e6dc/cartography/intel/aws/iam.py#L523-L524) to define the id of an AWSPolicy node (so that it differentiates between inline and managed policies). Can we hold off on tagging policies for a follow-up PR?